File: Platform/Workspace/Typescript/workspace/libs/workspace/protocol/src/pull/PullRequest.ts
import { Pull } from '@allors/workspace/data';

export class PullRequest {
  public pulls?: Pull[];

  constructor(fields?: Partial<PullRequest>) {
    Object.assign(this, fields);
  }

  public toJSON() {
    return {
      p: this.pulls,
    };
  }
}

File: Platform/Protocol/Allors.Protocol/Api/Pull/PullRequest.cs
// <copyright file="PullRequest.cs" company="Allors bv">
// Copyright (c) Allors bv. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Protocol.Remote.Pull
{
    using Allors.Protocol.Data;

    public class PullRequest
    {
        /// <summary>
        /// Pulls
        /// </summary>
        public Pull[] p { get; set; }
    }
}
